powa/static/js/utils/theme.js

```diff
@@ -0,0 +1,21 @@
+function checkTheme(vueInstance) {
+  const theme = localStorage.getItem("dark_theme");
+  if (theme) {
+    if (theme === "true") {
+      vueInstance.$vuetify.theme.dark = true;
+    } else {
+      vueInstance.$vuetify.theme.dark = false;
+    }
+  } else if (
+    window.matchMedia &&
+    window.matchMedia("(prefers-color-scheme: dark)").matches
+  ) {
+    vueInstance.$vuetify.theme.dark = true;
+    localStorage.setItem(
+      "dark_theme",
+      vueInstance.$vuetify.theme.dark.toString()
+    );
```

Comment:
In order to avoid storing a string in the localStorage, I'd recommend storing the information like this:

```js
localStorage.setItem("theme", {"dark": vueInstance.$vuetify.theme.dark});
```

Reply:
but setItem() take two strings parameters : `(method) Storage.setItem(key: string, value: string): void`

Reply:
All right, I forgot about that.